### PR TITLE
Show no index warning

### PR DESCRIPTION
**Pull request by Kashif-S**
Jira: GLT-2357

### CODE REVIEW

**Comment by sqwci** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [EditPoolController.java#L335](https://github.com/TGAC/miso-lims/blob/a069beb7a27d09365a7f4bf0b0e55c0ca3b45ee8/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L335): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)
1. ![MINOR][MINOR] [PoolImpl.java#L410](https://github.com/TGAC/miso-lims/blob/a069beb7a27d09365a7f4bf0b0e55c0ca3b45ee8/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/PoolImpl.java#L410): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
1. ![MINOR][MINOR] [PoolImpl.java#L596](https://github.com/TGAC/miso-lims/blob/a069beb7a27d09365a7f4bf0b0e55c0ca3b45ee8/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/PoolImpl.java#L596): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
1. ![MINOR][MINOR] [EditPoolController.java#L142](https://github.com/TGAC/miso-lims/blob/a069beb7a27d09365a7f4bf0b0e55c0ca3b45ee8/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L142): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L175](https://github.com/TGAC/miso-lims/blob/a069beb7a27d09365a7f4bf0b0e55c0ca3b45ee8/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L175): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L180](https://github.com/TGAC/miso-lims/blob/a069beb7a27d09365a7f4bf0b0e55c0ca3b45ee8/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L180): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L274](https://github.com/TGAC/miso-lims/blob/a069beb7a27d09365a7f4bf0b0e55c0ca3b45ee8/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L274): Replace "@RequestMapping(method = RequestMethod.POST)" with "@PostMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L290](https://github.com/TGAC/miso-lims/blob/a069beb7a27d09365a7f4bf0b0e55c0ca3b45ee8/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L290): Replace "@RequestMapping(method = RequestMethod.POST)" with "@PostMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L339](https://github.com/TGAC/miso-lims/blob/a069beb7a27d09365a7f4bf0b0e55c0ca3b45ee8/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L339): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/ListTablesIT.java [hunk 1]
```diff
@@ -291,6 +292,7 @@ public void testListPoolsPageSetup() throws Exception {
     testTabbedPageSetup(ListTarget.POOLS, poolsColumns);
   }
 
+  @Ignore // Broken since warnings are currently not sorted.
```
Comment (apmasell) [inline miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/ListTablesIT.java]: These two sorting tests are important. In the `list_x.js` files, set `bSortable` to false and make sure it's not on the sortable columns list.

**Comment by sqwci** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [EditPoolController.java#L335](https://github.com/TGAC/miso-lims/blob/c587b07825a3b98b6a4c3a0b161fc50865121397/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L335): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)
1. ![MINOR][MINOR] [PoolImpl.java#L410](https://github.com/TGAC/miso-lims/blob/c587b07825a3b98b6a4c3a0b161fc50865121397/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/PoolImpl.java#L410): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
1. ![MINOR][MINOR] [PoolImpl.java#L596](https://github.com/TGAC/miso-lims/blob/c587b07825a3b98b6a4c3a0b161fc50865121397/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/PoolImpl.java#L596): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
1. ![MINOR][MINOR] [EditPoolController.java#L142](https://github.com/TGAC/miso-lims/blob/c587b07825a3b98b6a4c3a0b161fc50865121397/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L142): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L175](https://github.com/TGAC/miso-lims/blob/c587b07825a3b98b6a4c3a0b161fc50865121397/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L175): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L180](https://github.com/TGAC/miso-lims/blob/c587b07825a3b98b6a4c3a0b161fc50865121397/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L180): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L274](https://github.com/TGAC/miso-lims/blob/c587b07825a3b98b6a4c3a0b161fc50865121397/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L274): Replace "@RequestMapping(method = RequestMethod.POST)" with "@PostMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L290](https://github.com/TGAC/miso-lims/blob/c587b07825a3b98b6a4c3a0b161fc50865121397/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L290): Replace "@RequestMapping(method = RequestMethod.POST)" with "@PostMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L339](https://github.com/TGAC/miso-lims/blob/c587b07825a3b98b6a4c3a0b161fc50865121397/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L339): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by sqwci** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [EditPoolController.java#L335](https://github.com/TGAC/miso-lims/blob/73b7614bb44dfef911ff4f17b7dcc4a06676776e/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L335): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)
1. ![MINOR][MINOR] [PoolImpl.java#L410](https://github.com/TGAC/miso-lims/blob/73b7614bb44dfef911ff4f17b7dcc4a06676776e/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/PoolImpl.java#L410): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
1. ![MINOR][MINOR] [PoolImpl.java#L596](https://github.com/TGAC/miso-lims/blob/73b7614bb44dfef911ff4f17b7dcc4a06676776e/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/PoolImpl.java#L596): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
1. ![MINOR][MINOR] [EditPoolController.java#L142](https://github.com/TGAC/miso-lims/blob/73b7614bb44dfef911ff4f17b7dcc4a06676776e/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L142): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L175](https://github.com/TGAC/miso-lims/blob/73b7614bb44dfef911ff4f17b7dcc4a06676776e/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L175): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L180](https://github.com/TGAC/miso-lims/blob/73b7614bb44dfef911ff4f17b7dcc4a06676776e/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L180): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L274](https://github.com/TGAC/miso-lims/blob/73b7614bb44dfef911ff4f17b7dcc4a06676776e/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L274): Replace "@RequestMapping(method = RequestMethod.POST)" with "@PostMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L290](https://github.com/TGAC/miso-lims/blob/73b7614bb44dfef911ff4f17b7dcc4a06676776e/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L290): Replace "@RequestMapping(method = RequestMethod.POST)" with "@PostMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L339](https://github.com/TGAC/miso-lims/blob/73b7614bb44dfef911ff4f17b7dcc4a06676776e/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L339): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/PoolPageIT.java [hunk 2]
```diff
@@ -277,6 +279,41 @@ public void testRemoveDilutions() {
...
+    PoolPage page = PoolPage.getForEdit(getDriver(), getBaseUrl(), id);
+    assertTrue("Page fails to show '" + warning + "' warning", page.getField(Field.WARNINGS).contains(warning));
+  }
+
+  @Test
+  public void testNoWarnings() {
+    PoolPage page = PoolPage.getForEdit(getDriver(), getBaseUrl(), 120001L);
+    DataTable table = page.getTable(PoolTableWrapperId.INCLUDED_DILUTIONS);
+    assertFalse(table.doesColumnContainSubstring(Columns.INDICES, "(NO INDEX)"));
+    assertFalse(table.doesColumnContainSubstring(Columns.INDICES, "(NEAR-DUPLICATE INDEX)"));
```
Comment (callunity) [inline miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/PoolPageIT.java]: This should be "INDICES" instead of "INDEX"

**Comment by sqwci** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [EditPoolController.java#L329](https://github.com/TGAC/miso-lims/blob/de016ff66b8fcfa903f4417a16c7edddabb123b3/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L329): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)
1. ![MINOR][MINOR] [PoolImpl.java#L410](https://github.com/TGAC/miso-lims/blob/de016ff66b8fcfa903f4417a16c7edddabb123b3/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/PoolImpl.java#L410): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
1. ![MINOR][MINOR] [PoolImpl.java#L596](https://github.com/TGAC/miso-lims/blob/de016ff66b8fcfa903f4417a16c7edddabb123b3/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/PoolImpl.java#L596): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
1. ![MINOR][MINOR] [EditPoolController.java#L141](https://github.com/TGAC/miso-lims/blob/de016ff66b8fcfa903f4417a16c7edddabb123b3/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L141): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L169](https://github.com/TGAC/miso-lims/blob/de016ff66b8fcfa903f4417a16c7edddabb123b3/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L169): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L174](https://github.com/TGAC/miso-lims/blob/de016ff66b8fcfa903f4417a16c7edddabb123b3/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L174): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L268](https://github.com/TGAC/miso-lims/blob/de016ff66b8fcfa903f4417a16c7edddabb123b3/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L268): Replace "@RequestMapping(method = RequestMethod.POST)" with "@PostMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L284](https://github.com/TGAC/miso-lims/blob/de016ff66b8fcfa903f4417a16c7edddabb123b3/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L284): Replace "@RequestMapping(method = RequestMethod.POST)" with "@PostMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L333](https://github.com/TGAC/miso-lims/blob/de016ff66b8fcfa903f4417a16c7edddabb123b3/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L333): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/PoolPageIT.java (hunk 2 above)
Comment (Kashif-S) [reply miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/PoolPageIT.java]: The Dilutions Included table displays warnings that refer to a singular index, so I think "INDEX" is correct here. It should, however, also be "INDEX" on the line below.

**Comment by sqwci** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [EditPoolController.java#L329](https://github.com/TGAC/miso-lims/blob/2e2d50b3698cff141bf2112553720e60470f1c0d/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L329): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)
1. ![MINOR][MINOR] [PoolImpl.java#L410](https://github.com/TGAC/miso-lims/blob/2e2d50b3698cff141bf2112553720e60470f1c0d/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/PoolImpl.java#L410): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
1. ![MINOR][MINOR] [PoolImpl.java#L596](https://github.com/TGAC/miso-lims/blob/2e2d50b3698cff141bf2112553720e60470f1c0d/core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/PoolImpl.java#L596): Replace this lambda with a method reference. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
1. ![MINOR][MINOR] [EditPoolController.java#L141](https://github.com/TGAC/miso-lims/blob/2e2d50b3698cff141bf2112553720e60470f1c0d/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L141): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L169](https://github.com/TGAC/miso-lims/blob/2e2d50b3698cff141bf2112553720e60470f1c0d/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L169): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L174](https://github.com/TGAC/miso-lims/blob/2e2d50b3698cff141bf2112553720e60470f1c0d/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L174): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L268](https://github.com/TGAC/miso-lims/blob/2e2d50b3698cff141bf2112553720e60470f1c0d/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L268): Replace "@RequestMapping(method = RequestMethod.POST)" with "@PostMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L284](https://github.com/TGAC/miso-lims/blob/2e2d50b3698cff141bf2112553720e60470f1c0d/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L284): Replace "@RequestMapping(method = RequestMethod.POST)" with "@PostMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)
1. ![MINOR][MINOR] [EditPoolController.java#L333](https://github.com/TGAC/miso-lims/blob/2e2d50b3698cff141bf2112553720e60470f1c0d/miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java#L333): Replace "@RequestMapping(method = RequestMethod.GET)" with "@GetMapping" [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4488)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**